SmartCarMQTTApp/app/src/main/java/com/example/smartcarmqttapp/PracticeTheoryActivity.java

```diff
@@ -3,26 +3,55 @@
 import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.app.Dialog;
 import android.content.Intent;
+import android.graphics.Color;
 import android.os.Bundle;
+import android.os.CountDownTimer;
 import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.CompoundButton;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.Switch;
 import android.widget.TextView;
+import android.widget.Toast;
 
 import com.google.android.material.bottomnavigation.BottomNavigationView;
 
 import org.w3c.dom.Text;
 
 import java.util.List;
+import java.util.Locale;
 
 public class PracticeTheoryActivity extends AppCompatActivity {
 
     private BottomNavigationView bottomNavigationView;
+    private Dialog settingsDialog;
+    private ImageView settingsButton;
+
+    private int questionCount = 0;
+
+    //Countdown timer
+    public static final int TEN_MIN_IN_MILLIS = 10000;
```

Comment:
This is ten seconds not ten minutes, I seem to have forgotten to change it back after testing